plugin.js

```diff
@@ -25,6 +33,16 @@ function fastifyCookieSetCookie (reply, name, value, options, signer) {
   }
 
   const serialized = cookie.serialize(name, value, opts)
+
+  const bytes = calculateCookieSize(serialized)
+
+  if (bytes >= 4096) {
+    const reachedOrExceeded = bytes > 4096 ? 'exceeded' : 'reached'
+    reply.server.log.warn(`Cookie[${name}] has ${reachedOrExceeded} safe size limit of 4096 bytes with current size of ${bytes} bytes`)
+  } else {
+    reply.server.log.debug(`Cookie[${name}] is now ${bytes} bytes of allowed 4096 bytes for safe limit`)
```

Comment:
Avoid template tags in log lines and use properties, e.g. `log.debug({ prop }, msg)`.

Reply:
Oh didn't know that was possible - but may I ask why? (I'll change it though)

Reply:
Template tagged strings are always evaluated. Providing a format string to the logger method will result in the string being evaluated only upon the logger method being invoked.

Reply:
Changed it, used combination of both now - uses the first param to add it to the JSON and uses a msg with `util.format` based formatters to get the message right